### PR TITLE
[MRG+1] Minor fixes

### PR DESCRIPTION
**Pull request by Eric89GXL**
Fixes a couple of minor bugs, one with `filter_chpi` and another with `filter_data` where sometimes we weren't returning the filtered data (!).

### CODE REVIEW

**Comment by wmvanvliet** [comment]
In exactly what usage scenario wasn't the filtered data returned?

File: mne/filter.py [hunk 1]
```diff
@@ -807,23 +814,19 @@ def filter_data(data, sfreq, l_freq, h_freq, picks=None, filter_length='auto',
         if l_freq == 0:
             l_freq = None
     if l_freq is None and h_freq is not None:
-        logger.info('Low-pass filtering at %0.2g Hz' % h_freq)
-        low_pass_filter(data, sfreq, h_freq,
-                        filter_length=filter_length,
-                        trans_bandwidth=h_trans_bandwidth, method=method,
-                        iir_params=iir_params, picks=picks, n_jobs=n_jobs,
-                        copy=copy, phase=phase, fir_window=fir_window)
+        data = low_pass_filter(
```
Comment (Eric89GXL) [inline mne/filter.py]: Here (low-pass in filter_data with copy=True)

File: mne/filter.py [hunk 2]
```diff
@@ -807,23 +814,19 @@ def filter_data(data, sfreq, l_freq, h_freq, picks=None, filter_length='auto',
...
+            data, sfreq, h_freq, filter_length=filter_length,
+            trans_bandwidth=h_trans_bandwidth, method=method,
+            iir_params=iir_params, picks=picks, n_jobs=n_jobs,
+            copy=copy, phase=phase, fir_window=fir_window)
     if l_freq is not None and h_freq is None:
-        logger.info('High-pass filtering at %0.2g Hz' % l_freq)
-        high_pass_filter(
+        data = high_pass_filter(
```
Comment (Eric89GXL) [inline mne/filter.py]: And here (high-pass in filter_data with copy=True)

**Comment by wmvanvliet** [comment]
ouch :)

**Comment by Eric89GXL** [comment]
Yeah no kidding. At least we haven't released with it

**Comment by Eric89GXL** [comment]
+1 for merge from you @wmvanvliet? If so I'll go ahead and merge since this one is pretty minor

**Comment by wmvanvliet** [comment]
yup, go ahead.

**Pull request merged**